components/bridge/Withdraw.tsx

```diff
@@ -86,14 +63,19 @@ const Withdraw: React.FC<{}> = () => {
 			const tokenId = tokenExist.isSome
 				? tokenExist.unwrap()
 				: await api.query.genericAsset.nextAssetId();
-
 			Object.values(bridgeBalances).map((token: any) => {
 				if (token.tokenId === tokenId.toString()) {
-					setTokenBalance(token.balance);
+					if (token.balance > Number(amount)) {
+						setEnoughBalance(true);
+					} else {
+						setEnoughBalance(false);
+					}
```

Comment:
Could collapse this group of if statement into `setEnoughBalance(token.balance > Number(amount))`